PROJ-177 - Mobility (#114)
SW805F20/report: sections/introduction/mobility.tex

\section{Mobility}\label{sec:mobility}
As a part of the study regulation it is emphasized that the system needs to be mobile.

> The essence of the project idea as described in \autoref{sec:projectidea}, is that the players are mobile and easily can use their smartphone and virtual reality headset as the viewable device to view the game.

Reviewer: andet ord end essence

> The essentiality of the project idea as described in \autoref{sec:projectidea}, is that the players are mobile and easily can use their smartphone and virtual reality headset as the viewable device to view the game.

Reviewer 2: ```suggestion
The essentiality of the project idea as described in \autoref{sec:projectidea}, is that the players are mobile and can use their smartphone and Google Cardboard to view the game.
```

Reviewer 2: Tror ikke det er et godt brug af essentiality.
En lidt spøjs sætning.

Author: Tror ikke at vi har introduceret google cardboard og smart phone på dette tidspunkt, så derfor har jeg skrevet det andet.

> To achieve this form of mobility it requires us to handle wireless communication between users and by calculating the location of the players by the use of sensors.

Reviewer 2: ```suggestion
To achieve this form of mobility, the system needs to handle wireless communication between users and calculate the location of players by the use of sensors.
```

\\\\

> We also aim towards having the entire game as being mobile, which means that you would be able to set up the game at different locations.

Reviewer: lyder lidt blown

> This can however be more difficult, as some type of location-based service has to be used, as well as the need for connection between users to communicate.

Reviewer 2: mærkelig sætning. used og need passer ikke rigtig sammen i samme sætning. skal måske prøve og formulere begge dele lidt ens

> Setting up the game at other locations also requires more knowledge about the game from the user, than the regular player otherwise would know, as the person has to set up the sensors at correct positions to measure the distance between the components.

Reviewer 2: hvilken location er ikke en other location ? har vi en fast location ?

> Setting up the game at other locations also requires more knowledge about the game from the user, than the regular player otherwise would know, as the person has to set up the sensors at correct positions to measure the distance between the components.

Reviewer 2: ```suggestion
Setting up the game at other locations also requires more knowledge about the game from the user, than the regular player would have, as the user has to set up the anchors at correct positions to measure the distance between the anchors.
```

Author: Vi har ikke introduceret anchors, så derfor benytter jeg sensors i stedet for anchors
